### ISSUE TITLE
'NoneType' object is not callable on image_uris

### ISSUE THREAD

**Issue by pyrometheous** [NandaScott/Scrython#55]
I've been playing around with the API to grab images of cards based on their name. Most of the time this works fine, however I've recently run into an issue with some of the cards. For this example, "Birgi, God of Storytelling" is the card that's throwing a NoneType error.

``` Python
card = scrython.cards.Named(fuzzy="Birgi, God of Storytelling")
response = card.image_uris(index=0, image_type='png')
print(response)
```

When I do this with another card, "Breeding Pool", I get the image url:
``` Python
card = scrython.cards.Named(fuzzy="Breeding Pool")
response = card.image_uris(index=0, image_type='png')
print(response)
```

I've been reviewing the API information on the scryfall website, however I can't seem to find any information about this error.

**Comment by NandaScott** [comment]
Hey there, sorry about this issue. I'm guessing that this stems from the fact that `image_uris` are based on the `layout` of a card. This maps out an easy way to find the images, however it means that if WoTC/Scryfall add a new layout my library is now out of date. I'll work today to get a new version out so this issue is fixed (until Wizards invents 3 sided cards).

**Comment by NandaScott** [comment]
Issue should be fixed in `1.8.1`. Let me know if it isn't and I'll open this back up.

**Comment by pyrometheous** [comment]
Looks like that did the trick, thank you!
